app/components/cases/submissions/decisionmaking-flow-selector.hbs

```diff
@@ -1,47 +1,67 @@
 <AuFormRow ...attributes>
   <AuLabel @required={{true}} @inline={{true}}>{{capitalize (t "case")}}</AuLabel>
-  <PowerSelect
-    @disabled={{@disableEdit}}
-    @search={{perform this.searchDecisionmakingFlows}}
-    @searchMessage={{t "search-on-case-name"}}
-    @searchEnabled={{true}}
-    @noMatchesMessage={{t "no-results-found"}}
-    @loadingMessage={{t "please-be-patient"}}
-    @selected={{this.decisionmakingFlowOrTitle}}
-    @onChange={{this.onSelectDecisionmakingFlow}}
-    class="au-u-1-1"
-    as |decisionmakingFlow|
-  >
-    {{or decisionmakingFlow.case.shortTitle decisionmakingFlow}}
-  </PowerSelect>
-  {{#unless @disableEdit}}
-    <AuButton
-      @skin="link"
-      @icon="add"
-      class="au-u-margin-top-small"
-      {{on "click" (queue (toggle "isAddingNewDecisionmakingFlow" this) this.setFocus)}}
-    >
-      {{t "add-new-case"}}
-    </AuButton>
-  {{/unless}}
+  <div class="au-c-inline-input-button">
```

Comment:
what is this class? can't seem to find it locally or in the kaleidos-css repo

Reply:
Will be changing this, the buttons are a bit off vertical center. Also the width of the input field is small.